### ISSUE TITLE
menu item missing a text alternative

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#3936]
The facebook, twitter, linked in sharing menu items are shows as an icon only without any alt-text.

![Image](https://github.com/BirkbeckCTP/janeway/assets/5330770/465e15ad-83ef-40c0-8534-b2609ca016ff)

https://www.iastatedigitalpress.com/farmreports/article/id/12984/

(and Twitter one probably needs to be updated!)
this also applies to other icons being used, such as the cite and print options.

![Image](https://github.com/BirkbeckCTP/janeway/assets/5330770/7434c903-6060-4cf8-904f-b22a3802f67f)

The images are missing alt-text, and as they are links, this also means the links have no text description as to where they are going (which was a separate site improve report on link missing a text alternative).

this includes the expanding menu icon for narrower screens:

![Image](https://github.com/BirkbeckCTP/janeway/assets/5330770/9eeecdfa-fed5-4173-97ed-899ca04ad0b9)

**Comment by ajrbyers** [comment]
- [ ] Check clean and material
